src/diffusers/utils/dummy_pt_objects.py

```diff
@@ -895,6 +910,21 @@ def from_pretrained(cls, *args, **kwargs):
         requires_backends(cls, ["torch"])
 
 
+class TransformerTemporalModel(metaclass=DummyObject):
```

Comment:
why this?

Reply:
It was missing, needed to be added for the test to pass.